keyboards/keebio/levinson/keymaps/ksamborski/keymap.c

```diff
@@ -0,0 +1,158 @@
+#include QMK_KEYBOARD_H
+
+enum layers {
+  _QW,
+  _LW,
+  _RS,
+  _AD,
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+  MKITPNK,
+  DYNAMIC_MACRO_RANGE
```

Comment:
The core Dynamic Macro implementation doesn't require using `DYNAMIC_MACRO_RANGE` in your `custom_keycodes` enum.

```suggestion
```

Reply:
Specifically, the dynamic macros was converted to a core feature, rather than something that needs importing. 

As such, the keycodes are now core keycodes, and don't need to be defined here.